controllers/blogPosts.js

```diff
@@ -0,0 +1,30 @@
+const postServices = require('../services/blogPosts');
+const { status, intServerError } = require('../Helpers/status&messages');
+
+const createPost = async (req, res) => {
+try {
+  console.log('to no contrller');
+  const { title, content, categoryIds } = req.body;
+  const { id: userId } = req.user;
+  
+  const newPost = await postServices.createPost(title, content, categoryIds, userId);
```

Comment:
aqui também, precisa passar um objeto pro createPost.
No meu eu n passei o categoryIds pra ele aqui. Acho que pros requisitos futuros vc vai precisar mandar ele la pra tabela de ligação entre os posts e as categorias
```suggestion
  const newPost = await postServices.createPost({title, content, categoryIds, userId});
```